che-editors.yaml

```diff
@@ -406,88 +406,6 @@ editors:
         attributes:
           ports:
             - exposedPort: 8080
-  - schemaVersion: 2.1.0
-    metadata:
-      name: che-incubator/intellij-community/2020.3.3
```

Comment:
@svor is there some deprecation policy we need to follow before removing an editor from the registry?
Or is it ok to just drop it? Considering that most likely it's not widely used.

Reply:
As far as i know we don't have deprecation policy for editors.